git-ftp

```diff
@@ -1082,6 +1127,19 @@ handle_fetch() {
 # ------------------------------------------------------------
 # Actions
 # ------------------------------------------------------------
+
+action_snapshot() {
+	
```

Comment:
I use vim as well. Maybe we should include the settings at the top of the file? Mine are:
```
set noexpandtab
set shiftwidth=8
set softtabstop=8
```